context/app/static/js/components/home/ExternalLink/style.ts

```diff
@@ -0,0 +1,23 @@
+import { styled } from '@mui/material/styles';
+
+const Flex = styled('div')(({ theme }) => ({
```

Comment:
Can this be a stack?

Reply:
I had converted this directly from the legacy styles, but have updated the implementation to make better use of the stack utility and have a more useful name than `Flex`.